### PR TITLE
Add license of html5shiv, cc #5500

### PR DESCRIPTION
**Pull request by lcd78706**
https://github.com/aFarkas/html5shiv/blob/master/MIT%20and%20GPL2%20licenses.md

### CODE REVIEW

**Comment by lcd78706** [comment]
@kennynaoh 
Please review this pull request. Thanks!

**Comment by kennynaoh** [comment]
@lcd78706 
What's the reason to close it.

**Comment by lcd78706** [comment]
@kennynaoh 
I add only one license, but it has two licenses.

**Comment by kennynaoh** [comment]
@lcd78706 
You fix it in this PR, not close it.

**Comment by kennynaoh** [comment]
@lcd78706 
Please take a look at this https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json#L161, and fix the license field

File: ajax/libs/html5shiv/package.json [hunk 1]
```diff
@@ -20,5 +20,9 @@
     "files": [
       "**/*"
     ]
-  }
+  },
+  "license": [
```
Comment (kennynaoh) [inline ajax/libs/html5shiv/package.json]: @lcd78706 
It should be `licenses` here.

**Comment by kennynaoh** [comment]
@lcd78706 , please ping me after you fix the problem

**Comment by lcd78706** [comment]
@kennynaoh 
I modified it.

**Comment by kennynaoh** [comment]
LGTM
ping @cdnjs/intern2

**Comment by pvnr0082t** [comment]
LGTM

**Pull request merged**